packages/vuetify/src/components/VBtn/VBtn.sass

```diff
@@ -241,6 +241,10 @@
 
   .v-btn__content
     opacity: 0
+  
+  .v-progress-circular 
+    width: 60% !important
```

Comment:
This needs to be done without **important** and should also be in `_variables.scss`